src/panels/AuditLogsFilterPanel/Fields/Fields.tsx

```diff
@@ -18,11 +17,9 @@ import type { FormFields } from "../types";
 export enum Label {
   AFTER = "After",
   BEFORE = "Before",
-  FACADE = "Facade",
   METHOD = "Method",
```

Comment:
The column in the `Audit logs` table is named `Facade Method`, so I was wondering if maybe, for consistency reasons, we can name the label here the same. Although, at the same time, we send this as `method` in the request, so I think it can be left as it is as well.

Reply:
I think naming the field the same as the table makes sense. I'll update it.